source/2019-10-18-the-ember-times-issue-120.md

```diff
@@ -14,13 +14,15 @@ READMORE
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## Ember Bootstrap 3.0.0 released :tada:
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Just a few days ago [Ember Bootstrap 3.0.0 was released](https://twitter.com/simonihmig/status/1182661298464739330). [Ember Bootstrap](https://www.ember-bootstrap.com) brings the Bootstrap library to Ember without requiring any of the Bootstrap JavaScript. Instead it rebuilds the dynamic features of Bootstrap with native Ember Components.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The UI library enters the Octane-era with this new major version: It introduces full support of angle bracket component invocation syntax including an updated documentation. All components have have been refactored to native classes.
+
+While dropping support for some very old versions of Ember it still supports 2.18 to ease the migration path. This has introduced some complexity because Ember 2.18 isn't supported by [native class polyfill](https://github.com/pzuraq/ember-native-class-polyfill). Additionally extending the componentes provided using good old `Object.extend()` as well as native classes syntax is considered part of addon's public API. This introduced another challenging problem as a class field can not be overwritten with a computed property. A [`defaultValue` decorator](https://github.com/kaliber5/ember-bootstrap/blob/v3.0.0/addon/utils/default-decorator.js) was required to prevent a `Cannot set property foo ... which has only a getter` error. You might want to have a look into [the discord discussion about this one](https://discordapp.com/channels/480462759797063690/486553598436573206/618527893009465349) for details. In the end it has needed six release candidates and 3 1/2 months until the new major has been proven to be stable.
```

Comment:
Jeldrik, I thought this paragraph is a bit long and might lose the audience's interest due to technical explanation. (The behind-the-scenes stories were really fascinating, though, and I appreciated your sharing them with us.)

I made a quick pass at how I might rewrite it. Would you mind taking a look and changing it to how you see fit? Thanks.
```suggestion
Ember Bootstrap 3.0 drops support for some very old versions of Ember, but still supports 2.18+ to ease the migration path. The commitment to backwards compatibility introduced some technical challenges because Ember 2.18 isn't supported by [native class polyfill](https://github.com/pzuraq/ember-native-class-polyfill).

In addition, the addon's public API had to support both classic and Glimmer (native class) components. This introduced another challenge since [a class field cannot be overwritten by a computed property](https://discordapp.com/channels/480462759797063690/486553598436573206/618527893009465349). A [`defaultValue` decorator](https://github.com/kaliber5/ember-bootstrap/blob/v3.0.0/addon/utils/default-decorator.js) was required to prevent a `Cannot set property foo ... which has only a getter` error.

In the end, Ember Bootstrap saw six release candidates and 3 1/2 months until its new major release has been proven to be stable.
```

Reply:
@ijlee2 I really like your rewrite. Thanks a lot. There was just a small misunderstanding about what consumers should be able to do. I've taken your suggestion as a basis and fixed that in https://github.com/ember-learn/ember-blog/pull/316/commits/91e7cfb8dc98f727a7093dcb1ca27a2e15201561. Would be awesome if you could have a quick look at that sentence. Will rebase afterwards.